RELEASING.md

```diff
@@ -76,6 +76,8 @@ release is announced.
 1. Push the tag.
 1. Create a GitHub release on the Calico repo, upload the release file (calico_node/release-<VERSION>.tgz), and post the updated
    release notes (see above for details).
+1. Run `make add_canonical_urls VERSION=vX.Y` to add canonical URLs to the metadata of the latest release files. Pass in the number of the new latest release via `VERSION`. Example: `make add_canonical_urls VERSION=v1.3`, where `1.3` is the new latest release.
```

Comment:
the use of the word `latest` here is misleading. I think you mean the release that was previously the latest but is now about to become previous. right?  (I'm not suggesting any chnage yet, just trying to make sure I understand)